scripts/pointSelection/combineLayers.jl

```diff
@@ -0,0 +1,53 @@
+using BiodiversityObservationNetworks
+using SimpleSDMLayers
+using JSON
+using Downloads
+using GeoArrays: read 
+using DataFrames
+using CSV
+
+# Read in input arguments and json
+outputFolder = ARGS[1]
+filepath = joinpath(outputFolder,"input.json")
+outputFilepath = joinpath(outputFolder,"data/")
+isdir(outputFilepath) || mkdir(outputFilepath)
+
+input = JSON.parsefile(filepath)
+
+# Assign json objects to variables
+print(input["layerdata"])
+layermat = CSV.read(IOBuffer(input["layerdata"][1]), DataFrame)
+print(layermat)
+
+targetbalance = convert(Vector{Float64}, input["targetbalance"]) # this is the same as α
+layer_col = layermat[:, :layer]
+keep_rows = startswith.(layermat[:, :layer], "https:")
+
+#read in layers
+function get_simplesdmlayer(path)
+    geoarray = read(path)
+    SimpleSDMPredictor(geoarray.A[:,:,begin])
+end
+
+temppath = Downloads.download.(layermat[keep_rows, 1])
+layers = BiodiversityObservationNetworks.stack(get_simplesdmlayer.(temppath))
+
+# get weights with weights for a layer in columns 
+W = Matrix(layermat[keep_rows, 2:end])
+
+#priority = SimpleSDMPredictor(squish(layers, convert(Array{Float64}, transpose(W)), targetbalance))
+priority = SimpleSDMPredictor(squish(layers, W, targetbalance))
+
+priority_path = joinpath(outputFilepath, "priority_map.tiff")
+###################
+print(priority_path)
+# write out the priority map
+geotiff(priority_path, priority)
```

Comment:
In R, they have to specify parameters like COG and COMPRESS=DEFLATE to make sure that the file is a cloud optimized geotiff and not a regular geotiff, and that it isn't huge. It it necessary with this api?

Reply:
this is a super easy fix we discussed but did not implement, we can merge this soon, see https://github.com/PoisotLab/SpeciesDistributionToolkit.jl/issues/160

Reply:
currently SDT.jl uses `COMPRESS=LZW` as a default, but this could possibly also be a kwarg